src/chunker/embedder.py

```
@@ -307,6 +321,159 @@ def embed_chunks(
+            # Separate empty and non-empty texts
+            non_empty_indices = [j for j, t in enumerate(batch) if t.strip()]
+            non_empty_texts = [batch[j] for j in non_empty_indices]
+
+            # Get embeddings for non-empty texts
+            embeddings_map = {}
+            if non_empty_texts:
+                response = self.client.embeddings.create(
+                    model=self.model_name,
+                    input=non_empty_texts
+                )
+                for idx, emb_data in zip(non_empty_indices, response.data):
```

Ruff (0.14.10)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)